test/basic.test.js

```diff
@@ -1,174 +1,183 @@
 'use strict'
 
-const { test } = require('tap')
+const { test } = require('node:test')
 const boot = require('..')
 
-test('boot an empty app', (t) => {
+test('boot an empty app', (t, done_) => {
```

Comment:
`done` is used - fine, but `done_` is not clear

```suggestion
test('boot an empty app', (t, testCompleted) => {
```